cookiecutter/environment.py

```diff
@@ -4,7 +4,7 @@
 from cookiecutter.exceptions import UnknownExtension
 
 
-class ExtensionLoaderMixin(object):
+class ExtensionLoaderMixin():
```

Comment:
```suggestion
class ExtensionLoaderMixin:
```

And same in other places